source/slang/slang-ir-use-uninitialized-values.cpp

```diff
@@ -146,8 +173,54 @@ namespace Slang
 
         return addresses;
     }
+    
+    static void checkCallUsage(List<IRInst*>& stores, List<IRInst*>& loads, IRCall* call, IRInst* inst)
+    {
+        IRInst* callee = call->getCallee();
+
+        // Resolve the actual function
+        IRFunc* ftn = nullptr;
+        IRFuncType* ftype = nullptr;
+        if (auto spec = as<IRSpecialize>(callee))
+            ftn = as<IRFunc>(resolveSpecialization(spec));
+        else if (auto fwd = as<IRForwardDifferentiate>(callee))
+            ftn = as<IRFunc>(fwd->getBaseFn());
+        else if (auto rev = as<IRBackwardDifferentiate>(callee))
+            ftn = as<IRFunc>(rev->getBaseFn());
+        else if (auto wit = as<IRLookupWitnessMethod>(callee))
+            ftype = as<IRFuncType>(callee->getFullType());
```

Comment:
Is this correct to use `callee` not `wit`?
```
ftype = as<IRFuncType>(wit->getFullType());
```

Reply:
The reason why I am asking is that `wit` is unused and the compiler prints a warning.